src/api/init.ts

```diff
@@ -149,6 +150,18 @@ export async function startApiServer(opts: {
     res.redirect(`/extended/v1/status`);
   });
 
+  // in case env is not production and url is not provided
+  if (apiDocumentationUrl) {
+    app.use('/doc', (req, res) => {
+      res.redirect(apiDocumentationUrl ?? '');
+    });
+  } else if (!isProdEnv) {
+    app.use('/doc', (req, res) => {
+      const apiDocumentationPath = path.join(__dirname + '../../../docs/.tmp/index.html');
```

Comment:
Is this directory included/available in the docker images?

Reply:
![image](https://user-images.githubusercontent.com/47666766/149090220-1f9e6cd3-c883-42d5-90dc-e83c2014766f.png)

I created a new docker image after running `npm run build:docs` and analyzed its directories. This path is valid and directory is included in the docker image.